diff --git a/main.py b/main.py
index 22ef3f2a..fae72ce7 100755
--- a/main.py
+++ b/main.py
@@ -3,5 +3,12 @@
 name."""
 import allenact.main
 
+import sys
+import os
+sys.path.append(os.path.join(os.path.abspath('.'), 'projects/ithor_A2SP'))
+print(os.path.join(os.path.abspath('.'), 'projects/ithor_A2SP'))
+
+
+
 if __name__ == "__main__":
     allenact.main.main()
diff --git a/projects/ithor_A2SP b/projects/ithor_A2SP
--- a/projects/ithor_A2SP
+++ b/projects/ithor_A2SP
@@ -1 +1 @@
-Subproject commit c234a1827742cca4ad591bb19c0282b566a54122
+Subproject commit c234a1827742cca4ad591bb19c0282b566a54122-dirty
